Chess-Challenge/src/My Bot/MyBot.cs

```diff
@@ -37,10 +37,14 @@ public Move Think(Board board, Timer timer)
             return bestMove;
         }
 
+        var rnd = new Random((int)DateTime.Now.Ticks);
```

Comment:
Why do we need it? I guess that default implementation of Random costructor already asumes some non-constant seed.
But maybe it will be a nice idea to make this randimzer static and not to create it on each move.

Reply:
Cool! I had no idea haw empty constructor is implemented! thanks :)